static/js/protein/page.tsx

```diff
@@ -100,22 +100,60 @@ export class Page extends React.Component<PagePropType, PageStateType> {
   }
 
   render(): JSX.Element {
+    const name = this.props.nodeName;
+    const result = name.split("_");
```

Comment:
since `.split()` doesn't modify the original string, you can just call the `.split()` method on `this.props.nodeName` in line 103 instead of creating 2 new variables. 

also add a comment about the format of this.props.nodeName and what will happen when the split happens